polygeist/tools/cgeist/Test/Verification/sycl/sycl_use_host_module.cpp

```diff
@@ -1,14 +1,18 @@
 // RUN: clang++ -fsycl -fsycl-device-only -Xcgeist \
 // RUN:     -sycl-use-host-module=%S/host_module.mlir -O0 -w -emit-mlir -o - %s \
+// RUN:     -fno-sycl-raise-host\
```

Comment:
Why are we using `-fno-sycl-raise-host` for `sycl_use_host_module.cpp` test?

Reply:
Because this test loads a different host module already in MLIR format: `-sycl-use-host-module=%S/host_module.mlir`. If we would not disable host raising, we would end up with two conflicting host modules.